packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -5,8 +5,15 @@
 import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
 import { escapeRegExp } from 'lodash';
 
+let language = 'en-US';
+
 Given("I'm running the example {string}", (example: string) => {
-  cy.visit(example);
+  cy.visit(example, {
+    // See: https://glebbahmutov.com/blog/cypress-tips-and-tricks/#control-navigatorlanguage
+    onBeforeLoad(win) {
+      Object.defineProperty(win.navigator, 'language', { value: language });
```

Comment:
This will come in handy if & when we need to change language in a `.feature` file.